client/thin-replica-client/src/thin_replica_client.cpp

```diff
@@ -993,7 +995,8 @@ void ThinReplicaClient::Subscribe(uint64_t block_id) {
   // Create and launch thread to stream updates from the servers and push them
   // into the queue.
   stop_subscription_thread_ = false;
-  subscription_thread_.reset(new thread(&ThinReplicaClient::receiveUpdates, this));
+  subscription_thread_.reset(
+      new thread(&ThinReplicaClient::receiveUpdatesWrapper, this, std::ref(config_->update_queue)));
```

Comment:
`config_` is a member variable of the ThinReplicaClient object. We invoke the wrapper function on an already instantiated ThinReplicaClient object which is `this`. Therefore, the function will have access to `config_` so that we don't need this parameter.

Reply:
Yes, updated.